packages/main/src/Badge.js

```diff
@@ -67,6 +67,7 @@ const metadata = {
 		 * @public
 		 */
 		"default": {
+			propertyName: "text",
```

Comment:
Please do not create a propertyName called "text", as this will cause issues in the retfofit, where we create a property named text for all slots of type "Node".